homework_4/bet_acceptance_api/internal/api/controllers/validators/betAcceptanceValidator.go

```diff
@@ -0,0 +1,43 @@
+package validators
+
+import "code-cadets-2021/homework_4/bet_acceptance_api/internal/api/controllers/models"
+
+// BetAcceptanceValidator validates event update requests.
+type BetAcceptanceValidator struct {
+	coefficientUpperBound float64
+	paymentLowerBound     float64
+	paymentUpperBound     float64
+}
+
+// NewBetAcceptanceValidator creates a new instance of BetAcceptanceValidator.
+func NewBetAcceptanceValidator(coeffUpperBound, paymentLowerBound, paymentUpperBound float64) *BetAcceptanceValidator {
+	return &BetAcceptanceValidator{
+		coefficientUpperBound: coeffUpperBound,
+		paymentLowerBound:     paymentLowerBound,
+		paymentUpperBound:     paymentUpperBound,
+	}
+}
+
+// BetAcceptanceIsValid checks if received bet is valid.
+// Id is non-default
+// Coefficient is non-default and lower than given upper bound
+// Payment is non-default and between given upper and lower bound
+func (b *BetAcceptanceValidator) BetAcceptanceIsValid(betAcceptanceRequestDto models.BetAcceptanceRequestDto) bool {
```

Comment:
I would maybe name this `BetIsValid`, but that's just personal preference.

Reply:
I like that one better too, mine is just too much.